h/buildext.py

```diff
@@ -35,13 +39,32 @@
 resolve = AssetResolver().resolve
 
 
-def build_extension_common(env, bundle_app=False):
+class Resolver(webassets.env.Resolver):
+    """A custom resolver for webassets which can resolve
```

Comment:
I'll fix this separately, but docstrings should be formatted like one of the following examples:

```
"""A single line docstring."""
```

```
"""
The first line of the docstring.

Further description in a following paragraph...
...blah blah blah.
"""
```

There is a third style which some people use but which I have a mild preference for avoiding:

```
"""The first line of the docstring.

Further description in a following paragraph...
...blah blah blah.
"""
```

This one is none of these three and doesn't format correctly.